Makefile

```diff
@@ -397,34 +397,35 @@ ci-clean:
 	podman $(PODMAN_REMOTE_ARGS) \
 	    image prune --all --filter="label=aro-*=true"
 
+# Define OUTPUT_DIR at the top, so it works locally and in Azure pipelines
+OUTPUT_DIR ?= $(shell pwd)
+
+# Define OUTPUT_DIR at the top, so it works locally and in Azure pipelines
+OUTPUT_DIR ?= $(shell pwd)
+
 .PHONY: ci-rp
 ci-rp: fix-macos-vendor
-	podman $(PODMAN_REMOTE_ARGS) \
-		build . \
+	docker build . $(DOCKER_BUILD_CI_ARGS) \
+		-f Dockerfile.ci-rp \
+		--ulimit=nofile=4096:4096 \
+		--build-arg REGISTRY=$(REGISTRY) \
+		--build-arg ARO_VERSION=$(VERSION) \
+		--no-cache=$(NO_CACHE) \
+		--target=builder \
+		-t $(LOCAL_ARO_RP_BUILD_IMAGE):$(VERSION)
+	docker build . $(DOCKER_BUILD_CI_ARGS) \
 		-f Dockerfile.ci-rp \
 		--ulimit=nofile=4096:4096 \
 		--build-arg REGISTRY=$(REGISTRY) \
 		--build-arg ARO_VERSION=$(VERSION) \
 		--no-cache=$(NO_CACHE) \
```

Comment:
I found Waldo! This second `--no-cache` is why the builds take so long! We should use the cache on the second build, or use `--cache-from=$(LOCAL_ARO_RP_BUILD_IMAGE):$(VERSION)` to halve the build times.

Reply:
Thanks for catching that! I've updated the Makefile to use --cache-from=$(LOCAL_ARO_RP_BUILD_IMAGE):$(VERSION) for the second build instead of --no-cache, which should speed up the build process by reusing cached layers